### PR TITLE
Commit 1 - HW Week 9

### PR DESCRIPTION
**Pull request by MarkKolenbrander**
- Added Log statements
- Tried to write test

### CODE REVIEW

File: app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        val result = sut.getNewsArticles()
+            .first()
+
+        assertEquals(CustomResult.Success(expectedListOfArticles), result)
+        coVerify (exactly = 0){ mockApiService.getArticles(API_TOKEN, Country.NL,Category.GENERAL ) }
+    }
+
+    @Test fun `a list of articles is returned`() = runBlocking {
+        val api = mockApiService
```
Comment (zlippard) [inline app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt:48]: nit, you don't need to recreate this variable. Using `mockApiService` should be sufficient.

File: app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        coEvery { api.getArticles(API_TOKEN, Country.NL, Category.GENERAL) } returns ArticlesResponse(
+            "status",1, listOf(Article(
+                Source("id","name","description","url", Category.GENERAL, Language.NL, Country.NL),
+                "author",
+                "title" , "description", "url", "urlImage", "published", "content",
+            )
+            )
+        )
+        val articles = api.getArticles(API_TOKEN,Country.NL, Category.GENERAL)
```
Comment (zlippard) [inline app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt:58]: Remember to make your test calls on the "subject under test".  In this case it's running the test on your mock.

**Review by zlippard — APPROVED** [review]

File: app/build.gradle [hunk 3]
```diff
@@ -61,6 +58,8 @@ dependencies {
 
     //Glide
     implementation 'com.github.bumptech.glide:glide:4.13.2'
+    androidTestImplementation project(path: ':app')
+    androidTestImplementation project(path: ':app')
```
[the remark this answers is not in the document]
Comment (MarkKolenbrander) [reply app/build.gradle:62]: Thanks, changed it

File: app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt (hunk 1 above)
Comment (MarkKolenbrander) [reply app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt:48]: Thanks, changed it

File: app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt (hunk 2 above)
Comment (MarkKolenbrander) [reply app/src/test/java/com/markkolenbrander/capstonenewsapp/NewsArticleRepoTests.kt:58]: I have to look in to this

**Pull request merged**